pkg/model/proto/model.proto

```diff
@@ -690,3 +692,18 @@ message GenericResponse {
     }
 }
 
+message CollectMetricsOpts {
```

Comment:
I didn't see where is this struct used, so please remove it.

Reply:
This is used in the POST call...that is in another PR.

Reply:
I think you can add it in another PR, because you will have to update this docs again.